### PR TITLE
Created friendship group dict structure 

### PR DESCRIPTION
**Pull request by d-smit**
Created with @kinianlo, @kariosk and @nuttamas

Answers UCL-RITS/rse-classwork-2020#8

### CODE REVIEW

**Comment by d-smit** [comment]
Answers UCL-RITS/rse-classwork-2020#8

**Review by umitozmen — COMMENTED** [review]
did same as our group. It doesn't recognize the friendships one to other but overall look neat.
it is good that you added a print function to see members of the dict

File: group.py [hunk 1]
```diff
@@ -1,5 +1,30 @@
-"""An example of how to represent a group of acquaintances in Python."""
+group = {
```
Comment (premal-varsani) [inline group.py:1]: Good implementation of the nested dictionary.

File: group.py [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+         'connection': {
+             'John':'cousin',
+             'Zalika':'landlord'
+         }}   
+}
 
-# Your code to go here...
-
-my_group =
+print(group.items())
```
Comment (premal-varsani) [inline group.py:30]: Show example to show one of the friends relation.

**Pull request closed without merging**